tests/examples/tokens/test_erc1155ownable_ETHtester.py

```diff
@@ -0,0 +1,316 @@
+import pytest
+# ERC1155 ownable, opensea compatible tests
+# @author Dr. Pixel (github: @Doc-Pixel)
+
+# constants - contract deployment
+CONTRACT_NAME = 'TEST 1155'
+CONTRACT_SYMBOL= 'T1155'
+CONTRACT_URI = 'https://mydomain.io/NFTdata/{id}'
+NEW_CONTRACT_URI = 'https://mynewdomain.io/NFTdata/{id}'
+ERC165_INTERFACE_ID = '0x01ffc9a7'
+ERC1155_INTERFACE_ID = '0xd9b67a26'
+ERC1155_INTERFACE_ID_METADATA = '0x0e89341c'
+ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
+
+# minting test lists
+mintBatch = [1,2,3,4,5,6,7,8,9,10]
+mintBatch2 = [11,12,13,14,15,16,17,19,19,20]
+minBatchSetOf10 = [1,1,1,1,1,1,1,1,1,1]
+mintConflictBatch = [1,2,3]
+
+@pytest.fixture
+def NFT_contract(get_contract, w3):
+    owner, a1, a2 = w3.eth.accounts[0:3]
+    with open("examples/tokens/ERC1155ownable.vy") as f:
+        code = f.read()
+    c = get_contract(code, *[CONTRACT_NAME, CONTRACT_SYMBOL, CONTRACT_URI])
+    return c
+
+##### test fixtures #####
+
+@pytest.fixture
+def test_mint(NFT_contract, w3, assert_tx_failed):
+    NFT_contract.mint(w3.eth.accounts[1], 1, 1, '')
```

Comment:
A thing I like to do in these tests is to do something like on [line 23](https://github.com/vyperlang/vyper/pull/2807/files#diff-deae716dbe8bb855518c838d005e3213140e643c6f84d651536546994335c1a1R23) where you get some addresses out so you can write `{"from": owner}` instead of `{"from": w3.eth.accounts[3]}`, which is much easier to follow

Reply:
Personally, I am used to using the w3.eth.accounts[3] approach because there will not be a doubt which account you are using for the call.
Line 23 might be some stray code from your erc721 example.

Would your approach be desired or required?

Reply:
desired, just makes it much easier to read for me

Reply:
Changed this.